tests/test_other.py

```diff
@@ -7004,13 +7004,45 @@ def test_binaryen_mem(self):
           else:
             assert parts[6] == str(expect_max)
 
+  def test_binaryen_ctors(self):
+    if SPIDERMONKEY_ENGINE not in JS_ENGINES: return self.skip('cannot run without spidermonkey')
```

Comment:
Actually I'm not sure, I remember this didn't work in v8, but am not on a machine right now where I can test that.

Another factor is that this test just needs a wasm-capable engine. So really it should just mean we need to avoid node.js, as sm and v8 should work.

Should I enable the test for v8?